test/src/e2e/network2.test.ts

```diff
@@ -27,6 +27,8 @@ describe("network2 nodes", function() {
         nodeA = new CodeChain();
         nodeB = new CodeChain();
         await Promise.all([nodeA.start(), nodeB.start()]);
+        nodeA.keepLogs();
```

Comment:
Should we always keep the logs?
As far as I remember, it makes the Travis upload these logs even though this test succeeds when other tests in the e2e directory fail.

Reply:
I created the second commit to test this PR. I'll remove this commit later.
https://github.com/CodeChain-io/foundry/pull/309/commits/31225289d5f6cdfd9208889722cf7a8d9dc14467